### DISCUSSION TITLE
Blockly as a visual editor for Rust and Bevy games

### DISCUSSION THREAD

**Discussion by erlend-sh** [bevyengine/bevy#589] [Ideas]
@lupyuen made a very promising Proof of Concept in the embedded space:

https://github.com/lupyuen/blockly-mynewt-rust

Blockly, an open source visual programming language by Google, can be exported to multiple supported languages. @lupyuen has demonstrated that it’s possible to export Blockly programs to Rust. He also demonstrates how seamlessly Blockly can be integrated into VS Code.

If someone is interested in pushing this further, let’s coordinate here.

**Comment by AbstractDimensions** [comment]
I want this! I cant code rust for shit (hence why im trying to find a project like this) but i'd be willing to help make this a reality
